.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# git config blame.ignoreRevsFile .git-blame-ignore-revs
+a9eab5b3efdde452591566fdc70061cf0832798c
```

Comment:
why ignore this commit?

Reply:
It's a mainly fix for Git Lens users that hides this in blame, otherwise this commit will show as the latest change for almost every line. Note that this ignore isn't auto-applied, users who want to ignore it would have to manually run the specified command in the comment.